src/components/application_manager/doc/SDL.SDD.ApplicationManager.dox

```diff
@@ -0,0 +1,76 @@
+/**
+\page application_manager Application Manager Detailed Design
+## Table of contents
```

Comment:
I think this information should be added to a different repository: https://github.com/smartdevicelink/sdl_core_documentation

Reply:
@JackLivio sdl_documentation contains the high-level Software Architecture design. 
This is rather a low-level detailed design, that actually generated by Doxygen from the code.
Such implementation details should not be in SAD (software architecture design), the should be in SDD.